src/clients/firewall_stage1_finish.rb

```diff
@@ -66,6 +69,14 @@ def main
           "when"  => [:installation, :autoinst]
         }
       when "Write"
+        # The Autoyast configuration is mainly done during the second stage but
+        # we need to open remote services if enabled by linuxrc (bsc#1080630)
+        if Mode.autoinst && SuSEFirewall4Network.IsInstalled
+          Builtins.y2milestone("Preparing proposal for autoconfiguration.")
+          SuSEFirewall4Network.prepare_proposal unless SuSEFirewallProposal.GetChangedByUser
+          adjust_configuration
```

Comment:
adjust_configuration is defined int SuSEFirewallProposal but we don't want to add the package to the software selection as we do for installation. 

https://github.com/yast/yast-network/blob/SLE-12-SP3/src/clients/firewall_stage1_proposal.rb#L401

So this will be AutoYaST specific, and of course will define it as private.

Also, take in account that this client has been completely rewritten for master.